src/libs/HttpUtils.js

```diff
@@ -32,15 +32,11 @@ let reconnectAppCancellationController = new AbortController();
  * @param {String} url
  * @param {String} [method]
  * @param {Object} [body]
- * @param {Boolean} [canCancel]
  * @param {String} [command]
  * @returns {Promise}
  */
-function processHTTPRequest(url, method = 'get', body = null, canCancel = true, command = '') {
-    let signal;
-    if (canCancel) {
-        signal = command === CONST.NETWORK.COMMAND.RECONNECT_APP ? reconnectAppCancellationController.signal : cancellationController.signal;
-    }
+function processHTTPRequest(url, method = 'get', body = null, command = '') {
+    const signal = command === CONST.NETWORK.COMMAND.RECONNECT_APP ? reconnectAppCancellationController.signal : cancellationController.signal;
```

Comment:
@neil-marcellini so I would need to check for the log command here as well and that would solve the other thread?

Reply:
Yeah leave the signal undefined it it's a Log request, that should prevent logs from being cancelled.

Reply:
Done